Moclobemide/Moclobemide-Report.md

```diff
@@ -0,0 +1,383 @@
+
+![010_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/001_3_3_1_Model_Building/010_plotTimeProfile.png)
+
+![011_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/001_3_3_1_Model_Building/011_plotTimeProfile.png)
+
+![012_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/001_3_3_1_Model_Building/012_plotTimeProfile.png)
+
+![013_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/001_3_3_1_Model_Building/013_plotTimeProfile.png)
+
+### 3.3.2 Model Verification
+
+![001_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/001_plotTimeProfile.png)
+
+![002_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/002_plotTimeProfile.png)
+
+![003_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/003_plotTimeProfile.png)
+
+![004_plotPopulationTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/004_plotPopulationTimeProfile.png)
+
+![005_plotPopulationTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/005_plotPopulationTimeProfile.png)
+
+# 4 Conclusion
+The developed PBPK model of moclobemide describes the observed concentration-time courses very well.
+
+As most of the  non-CYP2C19 metabolism is via FMO3 which is mainly expressed in the liver, non-CYP2C19 CL is modeled in the liver only.
+
+Given not enough data to evaluate inhibition of emerging moclobemide metabolites, auto-inhibition of moclobemide appears appropriate approximation. 
+# 5 References
+**Dingemanse 1998** Dingemanse J, Wallnöfer A, Gieschke R, Guentert T, Amrein R. Pharmacokinetic and pharmacodynamic interactions between fluoxetine and moclobemide in the investigation of development of the “serotonin syndrome”. Clin Pharmacol Ther. 1998;63(4):403-413.
+
+**DrugBank DB01171** (https://www.drugbank.ca/drugs/DB01171)
+
+**Gram 1995** Gram LF, Guentert TW, Grange S, Vistisen K, Brøsen K. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: A panel study. *Clinical Pharmacology & Therapeutics*. 1995 Jun;57(6):670–7.
+
+**Guentert 1990** Guentert TW, Tucker G, Korn A, Pfefen JP, Haefelfinger P, Schoerlin MP. Pharmacokinetics of moclobemide after single and multiple oral dosing with 150 milligrams 3 times daily for 15 days. Acta Psychiatr Scand. 1990;82(S360):91-93.
+
+**Hoskins 2001** Hoskins J, Shenfield G, Murray M, Gross A. Characterization of moclobemide N -oxidation in human liver microsomes. Xenobiotica. 2001 Jan;31(7):387–97.
+
+**IPCS ICHEM** Website: inchem.org/documents/pims/pharm/pim151.htm#SectionTitle:3.3%20%20Physical%20properties
```

Comment:
Link not properly rendered. Use the full link (http://...

Reply:
Fixed